tmail-backend/integration-tests/jmap/memory-jmap-integration-tests/src/test/java/com/linagora/tmail/james/MemoryLinagoraFilterStateChangeTest.java

```diff
@@ -0,0 +1,24 @@
+package com.linagora.tmail.james;
+
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.modules.TestJMAPServerModule;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linagora.tmail.james.app.MemoryConfiguration;
+import com.linagora.tmail.james.app.MemoryServer;
+import com.linagora.tmail.james.common.FilterTypeNameModule;
+import com.linagora.tmail.james.common.LinagoraFilterStateChangeTest;
+
+public class MemoryLinagoraFilterStateChangeTest implements LinagoraFilterStateChangeTest {
+    @RegisterExtension
+    static JamesServerExtension jamesServerExtension = new JamesServerBuilder<MemoryConfiguration>(tmpDir ->
+        MemoryConfiguration.builder()
+            .workingDirectory(tmpDir)
+            .configurationFromClasspath()
+            .build())
+        .server(configuration -> MemoryServer.createServer(configuration)
+            .overrideWith(new TestJMAPServerModule())
+            .overrideWith(new FilterTypeNameModule()))
```

Comment:
FilterTypeNameModule should be part of the top level module declaration, tests should not need to specify it.